src/client/runner/automation/playback/upload.js

```diff
@@ -0,0 +1,19 @@
+import hammerhead from '../../deps/hammerhead';
+import { arrayUtils } from '../../deps/testcafe-core';
+import { ActionCanNotFindFileToUpload } from '../../../../errors/test-run';
+
+
+export default class UploadAutomation {
+    constructor (element, paths) {
+        this.element = element;
+        this.paths    = paths;
+    }
+
+    run () {
+        return hammerhead.doUpload(this.element, this.paths)
+            .then(errs => {
+                if (errs.length)
+                    throw new ActionCanNotFindFileToUpload(arrayUtils.map(errs, err => err.path));
```

Comment:
automations are used by legacy api and by api v2. But here we raise the error from the new test-run
```js
import { ActionCanNotFindFileToUpload } from '../../../../errors/test-run';
```

Maybe we should just reject the promise with incorrect filePaths here? Then legacy api will catch it and create the legacy api error and client driver (command executor) will catch it and create the `ActionCanNotFindFileToUpload` error?

@inikulin what do you think?